mgmt/rest/v1/tribe.go

```diff
@@ -42,12 +42,16 @@ var (
 	ErrMemberNotFound        = errors.New("Member not found")
 )
 
+// Deprecated: Update to apiV2(https://github.com/intelsdi-x/snap-client-go).
```

Comment:
I added deprecation comments to any functions that incorporate apiV1... This is part of the rest-tribe module located under the rest/v1 src directory; wouldn't we want to deprecate this specific module based on its dependency to Rest API v1?